dyn_em/module_diffusion_em.F

```diff
@@ -8107,7 +8107,7 @@ SUBROUTINE vertical_diffusion_implicit(ru_tendf, rv_tendf, rw_tendf, rt_tendf,&
           DO  k = kts+1, ktf-1
             rdz_w = -g/dnw(k)/(c1h(k)*mu(i,j) + c2h(k))
             IF(config_flags%use_theta_m == 1)THEN
-                qfac = 1.+rvovrd*moist(i,k,j,P_QV)
+                qfac = 1.+rvovrd*moist(i,kts,j,P_QV)
```

Comment:
Jimy,
Now that is this not a function of the k index, this qfac assignment should be pulled out of the inner k-loop.